packages/docs-site/.vitepress/config.ts

```diff
@@ -163,6 +163,7 @@ export default defineConfig({
       },
       { text: "Documentation", link: "/docs/ref", activeMatch: "/docs/ref" },
       { text: "Try Penrose", link: "pathname:///try/index.html" },
+      { text: "Join Discord", link: "https://discord.gg/Y3K2kHxp2b" },
```

Comment:
Did you configure this to be a permanent link? (The default expiration time is one week.)

Reply:
Yep